<!DOCTYPE html>
<html>

  <head>
    <meta charset='utf-8'>
    <meta http-equiv="X-UA-Compatible" content="chrome=1">
    <meta name="description" content="CS502 : ">

    <link rel="stylesheet" type="text/css" media="screen" href="stylesheets/stylesheet.css">

    <title>CS502</title>
  </head>

  <body>

    <!-- HEADER -->
    <div id="header_wrap" class="outer">
        <header class="inner">
          <a id="forkme_banner" href="https://github.com/danzhao22/CS502">View on GitHub</a>

          <h1 id="project_title">CS502</h1>
            <a class="project_page" href="./index.html">Course Project</a>&nbsp&nbsp&nbsp&nbsp
            <a class="paper_summary" href="./paper_summary.html">Paper Summary</a>
          <h2 id="project_tagline"></h2>
              <a class="zip_download_link" href="https://github.com/danzhao22/CS502/zipball/master">Download this project as a .zip file</a>
              <a class="tar_download_link" href="https://github.com/danzhao22/CS502/tarball/master">Download this project as a tar.gz file</a>
            </section>
        </header>
    </div>

    <!-- MAIN CONTENT -->
    <div id="main_content_wrap" class="outer">
      
      <section id="main_content" class="inner">
        <p><strong>Timeline</strong></p>
        <p><strong>Feb 23</strong>: Paper <a href="./files/paper.pdf">Dissecting cancer heterogeneity with a probabilistic genotype-phenotype model</a> selected to discuss within the group. More information is available at the <a href="paper_summary.html">Paper Summary</a> page.</p>
        <p><strong>Feb 9</strong>: Datasets <a href="http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000425.v1.p1">Dataset-1: GCD</a> and <a href="http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000092.v1.p1">SAGE</a> are selected and analyzed.</p>
        <p><strong>Feb 2</strong>: Our website for CS502 Computational Biology is available online</p>
        <h3>
          <a id="welcome-to-github-pages" class="anchor" href="#welcome-to-github-pages" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Team Members.
        </h3>
        
        <p><a href="https://github.com/danzhao22">Dan Zhao</a> (danzhao3117@gmail.com) and <a href="https://github.com/jwzhanggy">Jiawei Zhang</a> (jwzhanggy@gmail.com)</p>
        
        <h3>
          <a id="welcome-to-github-pages" class="anchor" href="#welcome-to-github-pages" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Project Abstract
        </h3>
        
        <p>To examine the shared genetic risk variants of obesity and substance dependence, a genome-wide data of 1829 individuals was used. The single locus association analysis and mixed effect linear model were applied, the top significant SNP for obesity located in FTO gene, and the top significant SNP for substance dependence located in the OPRM1 gene. Then, the univariate and bivariate methods was conducted to estimate the heritability and genetic correlation. The heritability for obesity and substance dependence is 0.26 and 0.22 respectively. But in this dataset, there is no significant evidence of genetic correlation. Finally, the central nervous system annotation data and pleiotropy information was integrated to prioritizing GWAS results.</p>
        
        <img src="./images/example.png" alt="Table_1" style="width:600px;height:600px;">
        
        <h3>
          <a id="welcome-to-github-pages" class="anchor" href="#welcome-to-github-pages" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><a href="http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000425.v1.p1">Dataset-1: GCD</a>
        </h3>
        
        <p>The <a href="http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000425.v1.p1">GCD</a> samples include 2379 European Americans. All subjects were recruited for studies of the genetics of drug dependence or alcohol dependence. The sample consisted of small nuclear families originally collected for linkage studies, and unrelated individuals. Subjects were recruited at ﬁve US clinical sites: Yale University School of Medicine (APT Foundation, New Haven, CT, USA), the University of Connecticut Health Center (Farmington, CT, USA), the University of Pennsylvania Perelman School of Medicine (Philadelphia, PA, USA), the Medical University of South Carolina (Charleston, SC, USA) and McLean Hospital (Belmont, MA, USA). 14All subjects were interviewed using an electronic version of the semi-structured assessment for Drug Dependence and alcoholism (SSADDA)15 to derive diagnoses for major psychiatric traits according to DSM-IV (Diagnostic and Statistical Manual of Mental Disorders, 4th edition) criteria.</p>
        <h3>
          <a id="welcome-to-github-pages" class="anchor" href="#welcome-to-github-pages" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><a href="http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000092.v1.p1">Dataset-2: SAGE</a>
        </h3>
        <p>Another dataset included is the <a href="http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000092.v1.p1">SAGE</a> data (dbGaP study accession phs000092.v1.p1). In this study, case subjects were identified as having a lifetime history of alcohol dependence using DSM-IV criteria. Control subjects were required to report a history of drinking because alcohol use is required to develop alcohol dependence. Control subjects had no significant alcohol-dependence symptoms. Because of the genetic overlap between alcohol and drug dependence, a diagnosis of drug dependence was an exclusionary criterion for control subjects. The data recruited families with multiple members as well as unrelated individuals of European descent. The subjects came from 9 different sites across the U.S.</p>
        
        <h3>
          <a id="welcome-to-github-pages" class="anchor" href="#welcome-to-github-pages" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Genotyping and Quality Control
        </h3>
        <img src="./images/quality_control.png" alt="Quality_Control" style="width:600px;height:250px;">
        
        <p>Samples for GCD were genotyped on the Illumina HumanOmni1-Quad v1.0 microarray containing 988,306 autosomal SNPs. The samples in SAGE study were genotyped on the Illumina Human1M containing 1040107 SNPs. </p>

        <p>Sample quality control including removing related individuals, misidentified individuals, as well as individuals with genotype failure rate of 0.02. After sample quality control, the number of GCD samples reduced from 2379 to 1828, the number of SAGE samples reduced from 2668 to 1840.  ( Table 1.). </p>
        <img src="./images/table_1.png" alt="Table_1" style="width:700px;height:250px;">
        <p>For SNP quality control, SNPs with allele frequency > 1% were analyzed. A SNP call rate of 98% was required. Hardy–Weinberg equilibrium (HWE) was tested, and SNPs that deviated from HWE (P < 10−6) were excluded. After SNP quality control, the SNP number in GCD samples is 805782; the SNP number in SAGE samples is 845871.</p>
      </section>
    </div>
    
    <div id="footer_wrap" class="outer">
      <footer class="inner">
        <p class="copyright">CS502 maintained by <a href="https://github.com/danzhao22">danzhao22</a></p>
        <p>Published with <a href="https://pages.github.com">GitHub Pages</a></p>
      </footer>
    </div>
  </body>
</html>
