<!DOCTYPE html>
<html>

  <head>
    <meta charset='utf-8'>
    <meta http-equiv="X-UA-Compatible" content="chrome=1">
    <meta name="description" content="CS502 : ">

    <link rel="stylesheet" type="text/css" media="screen" href="stylesheets/stylesheet.css">

    <title>CS502</title>
  </head>

  <body>

    <!-- HEADER -->
    <div id="header_wrap" class="outer">
        <header class="inner">
          <a id="forkme_banner" href="https://github.com/danzhao22/CS502">View on GitHub</a>

          <h1 id="project_title">CS502</h1>
          <h2 id="project_tagline"></h2>

            < id="downloads">
              <a class="zip_download_link" href="https://github.com/danzhao22/CS502/zipball/master">Download this project as a .zip file</a>
              <a class="tar_download_link" href="https://github.com/danzhao22/CS502/tarball/master">Download this project as a tar.gz file</a>
            </section>
        </header>
    </div>

    <!-- MAIN CONTENT -->
    <div id="main_content_wrap" class="outer">
      <section id="main_content" class="inner">
        <h3>
          <a id="welcome-to-github-pages" class="anchor" href="#welcome-to-github-pages" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Team Members.
        </h3>
        
        <p><a href="https://github.com/danzhao22">Dan Zhao</a> (danzhao3117@gmail.com) and <a href="https://github.com/jwzhanggy">Jiawei Zhang</a> (jwzhanggy@gmail.com)</p>
        
        <h3>
          <a id="welcome-to-github-pages" class="anchor" href="#welcome-to-github-pages" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Project Abstract
        </h3>
        
        <p>To examine the shared genetic risk variants of obesity and substance dependence, a genome-wide data of 1829 individuals was used. The single locus association analysis and mixed effect linear model were applied, the top significant SNP for obesity located in FTO gene, and the top significant SNP for substance dependence located in the OPRM1 gene. Then, the univariate and bivariate methods was conducted to estimate the heritability and genetic correlation. The heritability for obesity and substance dependence is 0.26 and 0.22 respectively. But in this dataset, there is no significant evidence of genetic correlation. Finally, the central nervous system annotation data and pleiotropy information was integrated to prioritizing GWAS results.</p>
        
        <img src="./images/example.png" alt="Table_1" style="width:600px;height:600px;">
        
        <h3>
          <a id="welcome-to-github-pages" class="anchor" href="#welcome-to-github-pages" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Dataset-1: GCD
        </h3>
        
        <p>The GCD samples include 2379 European Americans. All subjects were recruited for studies of the genetics of drug dependence or alcohol dependence. The sample consisted of small nuclear families originally collected for linkage studies, and unrelated individuals. Subjects were recruited at ﬁve US clinical sites: Yale University School of Medicine (APT Foundation, New Haven, CT, USA), the University of Connecticut Health Center (Farmington, CT, USA), the University of Pennsylvania Perelman School of Medicine (Philadelphia, PA, USA), the Medical University of South Carolina (Charleston, SC, USA) and McLean Hospital (Belmont, MA, USA). 14All subjects were interviewed using an electronic version of the semi-structured assessment for Drug Dependence and alcoholism (SSADDA)15 to derive diagnoses for major psychiatric traits according to DSM-IV (Diagnostic and Statistical Manual of Mental Disorders, 4th edition) criteria.</p>
        <h3>
          <a id="welcome-to-github-pages" class="anchor" href="#welcome-to-github-pages" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Dataset-2: SAGE
        </h3>
        <p>Another dataset included is the SAGE data (dbGaP study accession phs000092.v1.p1). In this study, case subjects were identified as having a lifetime history of alcohol dependence using DSM-IV criteria. Control subjects were required to report a history of drinking because alcohol use is required to develop alcohol dependence. Control subjects had no significant alcohol-dependence symptoms. Because of the genetic overlap between alcohol and drug dependence, a diagnosis of drug dependence was an exclusionary criterion for control subjects. The data recruited families with multiple members as well as unrelated individuals of European descent. The subjects came from 9 different sites across the U.S.</p>
        
        <h3>
          <a id="welcome-to-github-pages" class="anchor" href="#welcome-to-github-pages" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Genotyping and Quality Control
        </h3>
        <img src="./images/quality_control.png" alt="Quality_Control" style="width:600px;height:250px;">
        
        <p>Samples for GCD were genotyped on the Illumina HumanOmni1-Quad v1.0 microarray containing 988,306 autosomal SNPs. The samples in SAGE study were genotyped on the Illumina Human1M containing 1040107 SNPs. </p>

        <p>Sample quality control including removing related individuals, misidentified individuals, as well as individuals with genotype failure rate of 0.02. After sample quality control, the number of GCD samples reduced from 2379 to 1828, the number of SAGE samples reduced from 2668 to 1840.  ( Table 1.). </p>
        <img src="./images/table_1.png" alt="Table_1" style="width:700px;height:250px;">
        <p>For SNP quality control, SNPs with allele frequency > 1% were analyzed. A SNP call rate of 98% was required. Hardy–Weinberg equilibrium (HWE) was tested, and SNPs that deviated from HWE (P < 10−6) were excluded. After SNP quality control, the SNP number in GCD samples is 805782; the SNP number in SAGE samples is 845871.</p>
      </section>
    </div>
    <!--
    <div id="main_content_wrap" class="outer">
      <section id="main_content" class="inner">
        <h3>
<a id="welcome-to-github-pages" class="anchor" href="#welcome-to-github-pages" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Welcome to GitHub Pages.</h3>

<p>This automatic page generator is the easiest way to create beautiful pages for all of your projects. Author your page content here <a href="https://guides.github.com/features/mastering-markdown/">using GitHub Flavored Markdown</a>, select a template crafted by a designer, and publish. After your page is generated, you can check out the new <code>gh-pages</code> branch locally. If you’re using GitHub Desktop, simply sync your repository and you’ll see the new branch.</p>

<h3>
<a id="designer-templates" class="anchor" href="#designer-templates" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Designer Templates</h3>

<p>We’ve crafted some handsome templates for you to use. Go ahead and click 'Continue to layouts' to browse through them. You can easily go back to edit your page before publishing. After publishing your page, you can revisit the page generator and switch to another theme. Your Page content will be preserved.</p>

<h3>
<a id="creating-pages-manually" class="anchor" href="#creating-pages-manually" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Creating pages manually</h3>

<p>If you prefer to not use the automatic generator, push a branch named <code>gh-pages</code> to your repository to create a page manually. In addition to supporting regular HTML content, GitHub Pages support Jekyll, a simple, blog aware static site generator. Jekyll makes it easy to create site-wide headers and footers without having to copy them across every page. It also offers intelligent blog support and other advanced templating features.</p>

<h3>
<a id="authors-and-contributors" class="anchor" href="#authors-and-contributors" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Authors and Contributors</h3>

<p>You can <a href="https://github.com/blog/821" class="user-mention">@mention</a> a GitHub username to generate a link to their profile. The resulting <code>&lt;a&gt;</code> element will link to the contributor’s GitHub Profile. For example: In 2007, Chris Wanstrath (<a href="https://github.com/defunkt" class="user-mention">@defunkt</a>), PJ Hyett (<a href="https://github.com/pjhyett" class="user-mention">@pjhyett</a>), and Tom Preston-Werner (<a href="https://github.com/mojombo" class="user-mention">@mojombo</a>) founded GitHub.</p>

<h3>
<a id="support-or-contact" class="anchor" href="#support-or-contact" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Support or Contact</h3>

<p>Having trouble with Pages? Check out our <a href="https://help.github.com/pages">documentation</a> or <a href="https://github.com/contact">contact support</a> and we’ll help you sort it out.</p>
      </section>
    </div>
-->
    <!-- FOOTER  -->
    <div id="footer_wrap" class="outer">
      <footer class="inner">
        <p class="copyright">CS502 maintained by <a href="https://github.com/danzhao22">danzhao22</a></p>
        <p>Published with <a href="https://pages.github.com">GitHub Pages</a></p>
      </footer>
    </div>

    

  </body>
</html>
