<!DOCTYPE html>
<html>

  <head>
    <meta charset='utf-8'>
    <meta http-equiv="X-UA-Compatible" content="chrome=1">
    <meta name="description" content="CS502 : ">

    <link rel="stylesheet" type="text/css" media="screen" href="stylesheets/stylesheet.css">

    <title>CS502</title>
  </head>

  <body>

    <!-- HEADER -->
    <div id="header_wrap" class="outer">
        <header class="inner">
          <a id="forkme_banner" href="https://github.com/danzhao22/CS502">View on GitHub</a>

          <h1 id="project_title">CS502</h1>
            <a class="project_page" href="./index.html"><strong>Course Project</strong></a>&nbsp&nbsp&nbsp&nbsp
            <a class="paper_summary" href="./paper_summary.html"><strong>Paper Summary</strong></a>
          <h2 id="project_tagline"></h2>
              <a class="zip_download_link" href="https://github.com/danzhao22/CS502/zipball/master">Download this project as a .zip file</a>
              <a class="tar_download_link" href="https://github.com/danzhao22/CS502/tarball/master">Download this project as a tar.gz file</a>
            </section>
        </header>
    </div>

    <!-- MAIN CONTENT -->
    <div id="main_content_wrap" class="outer">
      
      <section id="main_content" class="inner">
        <h3>
          <a id="welcome-to-github-pages" class="anchor" href="#welcome-to-github-pages" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Paper Summary
        </h3>
        <h4>
          <a id="welcome-to-github-pages" class="anchor" href="./files/paper.pdf" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link">Dissecting cancer heterogeneity with a probabilistic genotype-phenotype model</span></a>
        </h4>
        <p><strong>Authors</strong>: Dong-Yeon Cho and Teresa M. Przytycka</p>
        
        <!-- <h4>Abstract</h4>
        <p>One of the obstacles hindering a better understanding of cancer is its heterogeneity. However, computa- tional approaches to model cancer heterogeneity have lagged behind. To bridge this gap, we have de- veloped a new probabilistic approach that models in- dividual cancer cases as mixtures of subtypes. Our approach can be seen as a meta-model that summar- izes the results of a large number of alternative models. It does not assume predefined subtypes nor does it assume that such subtypes have to be sharply defined. Instead given a measure of phenotypic simi- larity between patients and a list of potential explana- tory features, such as mutations, copy number variation, microRNA levels, etc., it explains phenotypic similarities with the help of these features. We applied our approach to Glioblastoma Multiforme (GBM). The resulting model Prob_GBM, not only correctly inferred known relationships but also identified new properties underlining phenotypic similarities. The proposed probabilistic framework can be applied to model relations between similarity of gene expression and a broad spectrum of potential genetic causes.</p>
        -->
        
        <p><strong>Paper Summary</strong>: One of the obstacles hindering a better understanding of cancer is its heterogeneity. However, computational approaches to model cancer heterogeneity have lagged behind. To bridge this gap, they have developed a new probabilistic approach that models individual cancer cases as mixtures of subtypes. Their approach can be seen as a meta-model that summarizes the results of a large number of alternative models. It does not assume predefined subtypes nor does it assume that such subtypes have to be sharply defined. Instead given a measure of phenotypic similarity between patients and a list of potential explanatory features, such as mutations, copy number variation, microRNA levels, etc., it explains phenotypic similarities with the help of these features. They applied their approach to Glioblastoma Multiforme (GBM). The resulting model Prob_GBM, not only correctly inferred known relationships but also identified new properties underlining phenotypic similarities. The proposed probabilistic framework can be applied to model relations between similarity of gene expression and a broad spectrum of potential genetic causes.
        </p>
        
        <p><strong>Technique Summary</strong>: To construct the patient network, they use a unified gene expression profile obtained by applying factor analysis to genes present on three different microarray platforms.
        They confirmed that the top 5% cutoff provides reasonable modularity of the patient network by graphing the distribution of node clustering coefficients and overall network clustering coefficient at various cutoffs.
        As the explanatory features associated with nodes (patients), they consider copy number alternations, mutations and microRNA expression. Specifically, the model has one random variable per each gene and per each type of a genetic variation observed in this gene. They treat amplification and deletion as two different types of variations.
        They evaluated the stability of the learning algorithm for the model with different K.
        </p>
        <p><strong>Pros:</strong>: They not only confirmed known drivers of phenotypic differences but also identified novel ones and the relationships between them. By combining available information to generate an integrated result, Meta analysis seems reasonable and can save a considerable amount of resources. By combining studies, a meta-analysis increases the sample size and thus the power to study effects of interest.</p>
        <p><strong>Cons</strong>: In this study, due to the limitation of the dataset, many interesting variations of the model should be able to added in the future. For example, phenotype similarity might include survival time and responses to treatment. Features can be extended to include transcription factor biding, methylation, age, sex or environment. </br> Also, the main problem for meta analysis is that there is the potential for  bias and skewed data. It is important to pre-select the studies for meta analysis, ensuring that all of the research used is comparable across the various research programs, allowing a genuine statistical analysis.
        </p>
    </div>
    
    <div id="footer_wrap" class="outer">
      <footer class="inner">
        <p class="copyright">CS502 maintained by <a href="https://github.com/danzhao22">danzhao22</a></p>
        <p>Published with <a href="https://pages.github.com">GitHub Pages</a></p>
      </footer>
    </div>
  </body>
</html>
